downstream/modules/platform/proc-gw-team-access-resources.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-gw-team-access_{context}"]
+
+= Providing team access to a resource
+You can grant users access through the teams of which they are members. When you add a user as a member of a team, they inherit access to the roles and resources defined for that team.
```

Comment:
First sentence, line 6: I'd change to "You can grant users access based on their team membership."